pyscriptjs/src/stdio.ts

```diff
@@ -36,6 +38,42 @@ export class CaptureStdio implements Stdio {
     }
 }
 
+/** Stdio provider for sending output to DOM element
+ *  specified by ID. Used with "output" keyword.
+ *
+ */
+export class TargetedStdio implements Stdio{
+
+    target_id: string;
+
+    constructor(target_id: string) {
+        this.target_id = target_id;
+    }
+
+    stdout_writeline (msg: string) {
+        // Elements are identified by ID at write-time, not
+        // when constructed, in case the running script changes element IDs
+        const target = document.getElementById(this.target_id)
+        console.log(target)
+        if (target === null) { // No matching ID
+            createSingularWarning(`Output = "${this.target_id}" does not match the id of any element on the page.`, `Output = "${this.target_id}" does not match the id of any element on the page.`)
```

Comment:
Should these two params be the same?

It seems that `createSingularWarning` takes a `msg` and `elementName`

Reply:
This might be a terrible idea, but what do you think allowing the second param to be null and if its null we use the msg? 🤔 

But yeah after reading the rest of the code, it makes sense to be the same haha

Reply:
I don't hate it! I'll add that now, make the tweaks and add an explanatory docstring.

Reply:
Second argument to `createSingularWarning` is now optional. 😁

I also realized we didn't have any tests explicitly for the banner functionality (though createDeprecationWarning is used in many places in the tests, so we kinda did). Added some integration and unit tests.

Reply:
Awesome!

And good point, I as mostly just relying on the deprecated things to test the banner which is a bad idea haha  Thank you for creating tests for it!